packages/app-store/routing_forms/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "/*": "Don't modify slug - If required, do it using cli edit command",
+  "name": "Routing Forms",
+  "title": "Routing Forms",
+  "slug": "routing_forms",
+  "type": "routing_forms_other",
+  "imageSrc": "/api/app-store/routing_forms/icon.svg",
+  "logo": "/api/app-store/routing_forms/icon.svg",
+  "url": "https://cal.com/apps/routing_forms",
+  "variant": "other",
+  "categories": ["other"],
+  "publisher": "Cal.com",
+  "email": "help@cal.com",
+  "description": "It would allow a booker to connect with the right person or choose the right event, faster. It would work by taking inputs from the booker and using that data to route to the correct booker/event as configured by Cal user",
```

Comment:
Minor nit here @hariombalhara on the copy, I would avoid the word would as this will be implemented once the users see it so 
```diff
+"description": "Allows a booker to connect with the right person or choose the right event, faster. Works by taking inputs from the booker and using that data to route to the correct booker/event as configured by Cal user",
-"description": "It would allow a booker to connect with the right person or choose the right event, faster. It would work by taking inputs from the booker and using that data to route to the correct booker/event as configured by Cal user",
```